plugins/catalog/src/components/EntityProcessingErrorsPanel/EntityProcessingErrorsPanel.test.tsx

```diff
@@ -24,7 +24,7 @@ import {
 import { renderInTestApp, TestApiRegistry } from '@backstage/test-utils';
 import React from 'react';
 import { EntityProcessingErrorsPanel } from './EntityProcessingErrorsPanel';
-import { Entity, getEntityName } from '@backstage/catalog-model';
+import { AlphaEntity, getEntityName } from '@backstage/catalog-model';
```

Comment:
Shouldn't this be `@backstage/catalog-model/alpha`? as per the changeset?

Reply:
Not within the monorepo, gets tricky 😅, talked a bit about that in the PR body